### ISSUE TITLE
Number of resources, 211, is greater than maximum allowed, 200

### ISSUE THREAD

**Issue by cchapman900** [amplify-education/serverless-log-forwarding#26]
I just added this plugin to my `serverless.yml` and tried deploying, but got the error:
```
Error --------------------------------------------------
 
  The CloudFormation template is invalid: Template format error: Number of resources, 211, is greater than maximum allowed, 200
```

My service isn't that big though... I only have around 30 functions for about 4 related resources.

**Comment by jconstance-amplify** [comment]
Does the newly merged #27 fix this issue for you?

**Comment by rddimon** [comment]
Hi @cchapman900 

Hope you are doing well and you have already resolved that issue.
Closing this issue as there has been no response for a long time.

Please try our latest version and let us know in case of any issues.
